lerna.json

```diff
@@ -1,5 +1,5 @@
 {
-  "lerna": "2.0.0-beta.38",
+  "lerna": "2.1.2",
```

Comment:
is it needed in this PR?

Reply:
No. However, I think it's nice to get rid of the `beta` tag. Any reason to keep it?